runner-plugins/runner-scribe/main.go

```diff
@@ -17,9 +17,9 @@ import (
 	"strings"
 
 	"github.com/jvehent/gozdef"
-	"gopkg.in/gcfg.v1"
 	"github.com/mozilla/mig"
 	scribemod "github.com/mozilla/mig/modules/scribe"
+	"gopkg.in/gcfg.v1"
```

Comment:
Is this ordering change a necessary component of this commit, or merely aesthetic?

Reply:
This reorder was done automatically by `gofmt`, which prefers imports be listed alphabetically.